python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -264,14 +263,11 @@ def random_split(self, weights, seed=None):
 
         :return A list of Tables
         """
-        for w in weights:
-            if w < 0.0:
-                invalidInputError(False,
-                                  "Weights must be positive. Found weight value: %s" % w)
-        seed = seed if seed is not None else random.randint(0, sys.maxsize)
         df_array = self.df.randomSplit(weights, seed)
         return [self._clone(df) for df in df_array]
```

Comment:
should return a list or tuple?

Reply:
In random_split comment: `:return A list of Tables`